packages/create-block/lib/index.js

```diff
@@ -133,72 +134,59 @@ program
 					);
 
 					if ( ! variant && availableVariants.length > 1 ) {
-						const result = await inquirer.prompt( {
-							type: 'list',
-							name: 'variant',
+						variant = await select( {
 							message:
 								'The template variant to use for this block:',
-							choices: availableVariants,
+							choices: availableVariants.map( ( value ) => ( {
+								value,
+							} ) ),
 						} );
-						variant = result.variant;
 					}
 
 					const defaultValues = getDefaultValues(
-						pluginTemplate,
+						projectTemplate,
 						variant
 					);
 
-					const filterOptionsProvided = ( { name } ) =>
-						! Object.keys( optionsValues ).includes( name );
-					const blockPrompts = getPrompts(
-						pluginTemplate,
+					const blockAnswers = await runPrompts(
+						projectTemplate,
```

Comment:
I changed the name from `pluginTemplate` to `projectTemplate` as I realized that the name became confusing after we introduced the concept of scaffolding a block without a plugin. This part is used to scaffold only the block.  Consequently, the name `getPluginTemplate` became `getProjectTemplate`. It's also a step towards the potential introduction of the template for a theme that needs dev tools for JavaScript and CSS.